runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1309,6 +1314,12 @@ sub startRun {
 							if (!($endIntervalName =~ /InitialRamp\-(\d+)/)) {
 								my $tptStr = sprintf("%.2f", $statsSummary->{"throughput"});
 								my $rtStr = sprintf("%.2f", $statsSummary->{"avgRT"});
+								my $startUsersStr = $statsSummary->{"startActiveUsers"};
+								my $endUsersStr = $statsSummary->{"endActiveUsers"};
+								my $numRtOps = $statsSummary->{"totalNumRTOps"};
+								my $numFailedRt = $statsSummary->{"totalNumFailedRT"};
+								my $pctFailRTStr = sprintf("%.2f", 100 * ((1.0 * $numFailedRt) / $numRtOps));
+
```

Comment:
Should there be a zero check on numRtOps?

Reply:
Yes.  Fixed